opal/mca/btl/uct/btl_uct_tl.c

```diff
@@ -516,7 +516,11 @@ static int mca_btl_uct_evaluate_tl (mca_btl_uct_module_t *module, mca_btl_uct_tl
 	 * come up with a better estimate. */
 
 	/* UCT bandwidth is in bytes/sec, BTL is in MB/sec */
+#if UCT_API > UCT_VERSION(1, 7)
+	module->super.btl_bandwidth = (uint32_t) (MCA_BTL_UCT_TL_ATTR(tl, 0).bandwidth.dedicated / 1048576.0);
```

Comment:
need to take both dedicated and shared bw into account 
dedicated + shared/ppn

Reply:
fixed.